custom_components/frigate/switch.py

```diff
@@ -148,3 +158,10 @@ async def async_turn_off(self, **kwargs: Any) -> None:
             0,
             False,
         )
+
+    def _get_descriptive_name(self) -> str:
+        """Returns a more descriptive name for the switch."""
+        if self._switch_name == "audio":
```

Comment:
Rather than bake in audio-specific `if` to an otherwise "abstract" class, it may be better to pass in the switch description as an argument to `FrigateSwitch()` and then  not need this special casing.

Reply:
let me know what you think